tests/Api/Responses/shop/checkout/cart/update_cart.json

```diff
@@ -21,8 +23,10 @@
         "firstName": "Updated: Jane",
         "lastName": "Updated: Doe",
         "phoneNumber": "123456789",
+        "company": null,
```

Comment:
I think we can skip null values by setting `<value name="skip_null_values">true</value>`, just wondering if we need it actually

Reply:
This is the desired behavior, but I don't know why it doesn't work correctly in version 1.13.

Reply:
https://github.com/api-platform/core/issues/3600 It worked incorrectly in version 1.13